array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(num => num * 2);
+console.log('doubled:', doubled);
+
+const prices = numbers.map(num => `$${num.toString()}.00`);
+console.log('prices:', prices);
+
+const upperCased = languages.map(lang => lang.toUpperCase());
+console.log('upperCased:', upperCased);
+
+const firstLetters = languages.map(lang => lang.charAt(0));
```

Comment:
This works but remember that strings are array like, you can access a specific character in a string using bracket notation. `lang[0]` would produce the same result. No change required for this one, just FYI.